apps/file-editor/app/controllers/pages_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
 
     @pathname = Pathname.new(path)
 
-    if ! WhitelistPolicy.new(Rails.application.config.x.whitelist_paths).permitted?(@pathname)
+    if ! AllowlistPolicy.new(Rails.application.config.x.allowlist_paths).permitted?(@pathname)
```

Comment:
There's no class named `AllowlistPolicy`, that's why the tests are failing.

Reply:
The class was changed to `AllowlistPolicy` but it's probably because the file name didn't change.

Reply:
Yea apparently it's a rails thing. Anyhow, tests pass now so we're good.